ivy/functional/ivy/set.py

```diff
@@ -74,7 +74,34 @@ def unique_all(
         .. note::
            The order of unique elements is not specified and may vary between
            implementations.
-
+   
+    Examples
```

Comment:
please format the examples into different sections, which are functional and instance, with their respective subsections if applicable. Docstrings example samples are shown here (https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html) and there are multiple points listed down for reference, thanks!